src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionTests.cs

```diff
@@ -9259,59 +9259,139 @@ static class E
         }
 
         [Fact]
-        public void CircularImplicitConversions()
```

Comment:
Something strange with the merge. The test was not deleted, but the diff shows as it was.

---
In reply to: [123671280](https://github.com/dotnet/roslyn/pull/20337#discussion_r123671280) [](ancestors = 123671280)